### PR TITLE
[bugfix][kotlin-wiremock] fix problems with range responses 

### PR DESCRIPTION
**Pull request by stefankoppier**
Fixes issue #19305.

@dr4ke616 (2018/08) @karismann (2019/03) @Zomzog (2019/04) @andrewemery (2019/10) @4brunu (2019/11) @yutaka0m (2020/03) @stefankoppier (2022/06)

### PR checklist
 
- [X] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [X] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [X] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh ./bin/configs/*.yaml
  ./bin/utils/export_docs_generators.sh
  ``` 
  (For Windows users, please run the script in [Git BASH](https://gitforwindows.org/))
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  IMPORTANT: Do **NOT** purge/delete any folders/files (e.g. tests) when regenerating the samples as manually written tests may be removed.
- [X] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (upcoming 7.6.0 minor release - breaking changes with fallbacks), `8.0.x` (breaking changes without fallbacks)
- [X] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

File: .github/workflows/samples-kotlin-wiremock.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
       matrix:
         sample:
           - samples/server/petstore/kotlin-wiremock
+          - samples/server/petstore/kotlin-wiremock-responses
```
Comment (wing328) [inline .github/workflows/samples-kotlin-wiremock.yaml:22]: i think you will need to update line 6 to 
```
      - samples/server/petstore/kotlin-wiremock*/**
```

Comment (stefankoppier) [reply .github/workflows/samples-kotlin-wiremock.yaml:22]: Agreed, fix is pushed

Comment (wing328) [reply .github/workflows/samples-kotlin-wiremock.yaml:22]: 👍 

will merge when all tests pass

**Comment by wing328** [comment]
https://github.com/OpenAPITools/openapi-generator/actions/runs/10281937668/job/28452695469?pr=19309 not related to this PR

**Comment by stefankoppier** [comment]
> https://github.com/OpenAPITools/openapi-generator/actions/runs/10281937668/job/28452695469?pr=19309 not related to this PR

Ah, I was wondering why I didn't get any new samples locally ;) thanks

**Pull request merged**